src/promote-play-release.js

```diff
@@ -11,6 +11,8 @@ export async function run() {
     const rawServiceAccountJson = getInput('service-account-json-raw', { required: true });
     const inAppUpdatePriorityInput = getInput('inapp-update-priority');
     const userFractionInput = getInput('user-fraction');
+    const fromTrack = getInput('from-track') ?? 'beta';
```

Comment:
alternative names are `source-track` and `target-track` maybe? from/to felt more natural with the "promotion" concept though